Docs/source/running_cpp/parameters.rst

```diff
@@ -44,6 +44,18 @@ Overall simulation parameters
 * ``warpx.verbose`` (`0` or `1`)
     Controls how much information is printed to the terminal, when running WarpX.
 
+* ``warpx.random_seed`` (`string`) optional
+    If provided ``warpx.random_seed = random``, the random seed will be determined
+    using `std::clock()`, thus every simulation run produces different random numbers.
+    If provided ``warpx.random_seed = n``, and `n >= 0`,
+    the random_seed will be reset based on `n`.
```

Comment:
Then, if users give 1, it would produce the same result as the default random seed.
I feel like users may waste time running two simulations with the same random seed, then realize they should start from 2 to change the random seed, which is not very nice.
That's why I mentioned "Plus 2 here is to make sure that, any nonnegative integer random seed provided by the user will not produce the same result as the default random seed."

So, how about giving `the random_seed for each MPI rank is mpi_rank * (n+1).` and force users to start from 1?

Reply:
Sure, or just ask the user to specify n >= 2, even simpler.

Reply:
Actually I find it nice to let the user use `1` to run the default. Maybe we could just specify it in the doc.

Reply:
If I may jump in, I agree that it would be nice to let the user use 1 (the default value) and this default value should be specified in the documentation. This makes what's happening more clear and it avoids hiding details.

Reply:
@MaxThevenet @dpgrote Thanks for the comments, I will do that!